Packs/CommonPlaybooks/Playbooks/playbook-Block_Email_-_Generic.yml

```diff
@@ -41,40 +44,39 @@ tasks:
       type: title
       iscommand: false
       brand: ""
-      description: ''
     separatecontext: false
+    continueonerrortype: ""
     view: |-
       {
         "position": {
-          "x": 450,
-          "y": 440
+          "x": 265,
+          "y": 370
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
```

Comment:
Should also be true. If someone uses the generic playbook and doesn't have one of the packs of the subplaybooks it will cause it to fail for them